docs/docs/integrations/scim.mdx

```diff
@@ -19,12 +19,15 @@ GrowthBook's SCIM integration currently offers the following features:
 
 - User provisioning
 - User deprovisioning
+- Group Push
 
 When a user is provisioned, they are added to your GrowthBook organization with your organization's default role. After provisioning, admin users can adjust their roles and permissions via the GrowthBook application as needed. It's important to note that if a user is provisioned through SCIM, they can only be deprovisioned through your identity provider.
 
+When a group is added to "Push Groups" in your Okta SCIM application, groups and their members will be synced with GrowthBook via a corresponding Team (learn more). Please note that only members that have been added to GrowthBook will be added to the corresponding Team in GrowthBook. Similarly to user provisioning, new Teams will be created with your organization's default role and permissions for the Team will need to be adjusted within GrowthBook. Group removal or membership changes will need to be done through your identity provider.
```

Comment:
I think we may want to re-word this sentence

```Please note that only members that have been added to GrowthBook will be added to the corresponding Team in GrowthBook```

I think what this is saying is: only members that have been provisioned into the GrowthBook App will be added to the `Team` in GrowthBook. Maybe we just need an example of this?